src/NuGet.Services.Staging.Manager/Controllers/ViewBuilder.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Services.Staging.Database.Models;
+using NuGet.Client.Staging;
+
+namespace NuGet.Services.Staging.Manager.Controllers
+{
+    public static class ViewBuilder
+    {
+        public static StageListView CreateStageListView(Stage stage, StageMembership membership, string baseAddress)
+        {
+            var view = new StageListView();
+            ((StageView)view).Populate(stage, baseAddress);

```

Comment:
changed Populate from extension method to a regular method

---
In reply to: [66829943](https://github.com/NuGet/NuGet.Staging/pull/62#discussion_r66829943) [](ancestors = 66829943)